EKF/control.cpp

```diff
@@ -358,14 +358,12 @@ void Ekf::controlOpticalFlowFusion()
 		const bool is_quality_good = (_flow_sample_delayed.quality >= _params.flow_qual_min);
 		const bool is_magnitude_good = !_flow_sample_delayed.flow_xy_rad.longerThan(_flow_sample_delayed.dt * _flow_max_rate);
 		const bool is_tilt_good = (_R_to_earth(2, 2) > _params.range_cos_max_tilt);
-		const bool is_body_rate_comp_available = calcOptFlowBodyRateComp();
 
-		// check if enough integration time and fail if integration time is less than 50%
-		// of min arrival interval because too much data is being lost
-		const float delta_time_min = 0.5e-6f * (float)_min_obs_interval_us;
-		const float delta_time_max = 0.2f;
+		const float delta_time_min = fmaxf(0.8f * _delta_time_of, 0.001f);
```

Comment:
I feel that we should add unit test for these time checks. So that we exactly capture the behavior that we want.

Reply:
Yes, I will add tests for that